inference-engine/src/mkldnn_plugin/nodes/region_yolo.cpp

```diff
@@ -226,27 +308,61 @@ class RegionYoloImpl: public ExtLayerBase {
             end_index = IW * IH * (classes + 1);
             num_ = mask_size;
         }
-        int inputs_size = IH * IW * num_ * (classes + coords + 1);
-        int total_size = 2 * IH * IW;
+        size_t inputs_size = IH * IW * num_ * (classes + coords + 1);
+        size_t total_size = 2 * IH * IW;
+
+        const auto *src_data = inputs[0]->cbuffer().as<const uint8_t *>();
+        auto *dst_data = outputs[0]->buffer().as<uint8_t *>();
+
+        if (input_prec == output_prec) {
```

Comment:
Lets reuse `cpu_convert` here?

Reply:
Great idea! Thank you. Implemented in 50ed2b53c83877bcd115de768936d1ac76a3597d.